openmc/settings.py

```diff
@@ -188,7 +188,7 @@ class Settings:
         Kelvin. The value for 'method' should be 'nearest' or 'interpolation'.
         If the method is 'nearest', 'tolerance' indicates a range of temperature
         within which cross sections may be used. The value for 'range' should be
-        a pair a minimum and maximum temperatures which are used to indicate
+        a pair of a minimum and maximum temperature which are used to indicate
         that cross sections be loaded at all temperatures within the
```

Comment:
small nitpick I think this would read best
```The value for 'range' should be
        a pair of minimum and maximum temperatures which are used to indicate ...
```

Reply:
I like this better